gms/gossiper.cc

```diff
@@ -857,44 +857,51 @@ future<> gossiper::failure_detector_loop() {
 
 void gossiper::replicate_live_endpoints_on_change() {
     assert(this_shard_id() == 0);
-    //
-    // Gossiper task runs only on CPU0:
-    //
-    //    - If endpoint_state_map or _live_endpoints have changed - replicate
-    //      them across all other shards.
-    //    - Reschedule the gossiper only after execution on all nodes is done.
-    //
-    bool live_endpoint_changed = (_live_endpoints != _shadow_live_endpoints);
-    bool unreachable_endpoint_changed = (_unreachable_endpoints != _shadow_unreachable_endpoints);
+    (void)seastar::async([this] {
```

Comment:
Running this in the background is missing the entire point.

`replicate_live_endpoints_on_change` is supposed to **guarantee** that when it returns, live_endpoints were replicated. It obviously cannot guarantee if the replication happens in the background, no?